modules/ROOT/pages/split-aggregate-scope.adoc

```diff
@@ -0,0 +1,108 @@
+= Split Aggregate Scope
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule, split, aggregate, scope
+
+The Split Aggregate scope splits a collection and processes each part with the components within it in parallel (subject to any limitation in concurrency that has been configured). The output will be a collection with the result of each route. After all messages have been processed, the results will be aggregated before continuing with the flow in the same order they were before splitting the defined collection. 
+
+The Split Aggregate scope can be configured through the following fields described in this table:
+
+[%header,cols="1,3"]
+|===
+|Child element |Description
+|Collection (`collection`) | Specifies the expression that defines the collection of parts to be processed in parallel. By default, it will use the incoming payload.
+|==
+
+[%header,cols="1,3"]
+|===
+|Attribute |Description
+|Timeout (`timeout`) | Specifies the timeout for each parallel route. By default there is no timeout.
+|Max Concurrency (`maxConcurrency`) | Specifies the maximum level of parallelism that will be used by this router. By default, all routes will run in parallel.
+|Target Variable (`target`) | Specifies the variable where to save the processed payload. By default the output will be saved in the flow's payload. 
+|Target Value (`targetValue`) | Specifies an expression to evaluate against the operation's output value. The outcome of this expression will be stored in the target variable. By default, this will be the same as the operation's output value.
+|===
+
+This XML example adds to every element in the collection the string `"-result"`: 
+
+----
+<split-aggregate>
+	<collection>
+		#[['apple', 'banana', 'orange']]
+	</collection>
+	<set-payload value="#[payload ++ '-result']"/>
+</split-aggregate>
+
+----
+
+Every route starts with the same initial variable values. If a variables is modified within an specific route, this will not affect other routes. If a variable is added or modified in one route then, after aggregation, the value will be the defined by that route. If a variable is added or modified by more than one route then the  value will be a list of all the values defined for that variable within all the routes. 
```

Comment:
```suggestion
Every route starts with the same initial variable values. If a variables is modified within a specific route, this will not affect other routes. If a variable is added or modified in one route then, after aggregation, the value will be the defined by that route. If a variable is added or modified by more than one route then the  value will be a list of all the values defined for that variable within all the routes. 
```